projects/swift-protobuf/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update && apt install -y wget \
+           binutils \
+           libc6-dev \
+           libcurl3 \
+           libedit2 \
+           libgcc-5-dev \
+           libpython2.7 \
+           libsqlite3-0 \
+           libstdc++-5-dev \
+           libxml2 \
+           pkg-config \
+           tzdata \
+           zlib1g-dev
+
+RUN wget https://swift.org/builds/swift-5.3.3-release/ubuntu1604/swift-5.3.3-RELEASE/swift-5.3.3-RELEASE-ubuntu16.04.tar.gz
```

Comment:
I know nothing about docker, can this be set up to float on the "swift-latest" in some way to it won't need updates on a regular basis?  https://swift.org/download/#docker

Reply:
Not sure about docker things...
Could we replace 5.3.3 by latest in the url ? (does not work now)

Reply:
Maybe start with the Swift image instead of the oss-fuzz-base one?

Reply:
that is a question for someone in the oss-fuzz team, @inferno-chromium ?

Reply:
Using an image other than the OSS-Fuzz one will likely lead to many issues.

Are you also planning to integrate more swift libraries? If so, perhaps we want to start looking at how we can better integrate this into our base images and build scripts.

Reply:
> Are you also planning to integrate more swift libraries?

I guess so.
I was thinking to get the first POC running before generalizing ;-)